program-libs/account-checks/docs/ERRORS.md

```
@@ -0,0 +1,153 @@
+// check_discriminator returns InvalidDiscriminator (12006) or InvalidAccountSize (12010)
+check_discriminator::<MyType>(data)?;
+
+// check_pda_seeds returns InvalidSeeds (12016)
+check_pda_seeds(seeds, program_id, account)?;
+```
+
+## Error Messages in Logs
+
+Errors appear in transaction logs with their numeric codes:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)